### PR TITLE
docs: remove adapter examples

### PR DESCRIPTION
**Pull request by meesvandongen**
## Type

- [x] Documentation Update

## Description

Remove adapter usage from examples

## Related

- Related Issue https://github.com/Effect-TS/effect/pull/2602

### CODE REVIEW

**Comment by IMax153** [comment]
@mikearnaldi - does this PR need to wait until we upgrade the monorepo to TS 5.5?

**Comment by mikearnaldi** [comment]
> @mikearnaldi - does this PR need to wait until we upgrade the monorepo to TS 5.5?

No we don't really mind, from user perspective we assume 5.5

File: packages/sql-sqlite-bun/examples/Client.test.ts [hunk 1]
```diff
@@ -4,24 +4,24 @@ import { SqliteClient } from "@effect/sql-sqlite-bun"
...
-      let rows = yield* _(sql`SELECT * FROM test`)
+    Effect.gen(function*() {
+      const sql = yield* makeClient
+      yield* sql`CREATE TABLE test (id INTEGER PRIMARY KEY, name TEXT)`
+      yield* sql`INSERT INTO test (name) VALUES ('hello')`
+      let rows = yield* sql`SELECT * FROM test`
       expect(rows).toEqual([{ id: 1, name: "hello" }])
-      yield* _(sql`INSERT INTO test (name) VALUES ('world')`, sql.withTransaction)
-      rows = yield* _(sql`SELECT * FROM test`)
+      yield* sql`INSERT INTO test (name) VALUES ('world')`, sql.withTransaction
```
Comment (tim-smart) [inline packages/sql-sqlite-bun/examples/Client.test.ts:23]: Missing a pipe here :)

**Review by tim-smart — APPROVED** [review]
👍🏼

**Pull request merged**